script.js

```diff
@@ -0,0 +1,19 @@
+const name1 = "Joshua";
+const name2 = "Troy";
+const name3 = "Aaron";
+
+if (name1.length > name2.length && name1.length > name3.length)  {
+    console.log(`${name1} has the longest name.`);
+    }   else if (name2.length > name1.length && name2.length > name3.length)    {
```

Comment:
Your lines are indented a bit far to the right. If statements and the closing curly brackets line up on the left like this: 

if (name1.length > name2.length && name1.length > name3.length) {
  console.log(`${name1} has the longest name.`);
} else if (name2.length > name1.length && name2.length > name3.length) {
  console.log(`${name2} has the longest name.`);
} else if (name3.length > name1.length && name3.length > name2.length) {
  console.log(`${name3} has the longest name.`);
} etc...

Reply:
The console statements are supposed to be indented, but Github comment is removing my white space.